src/hooks.server.ts

```diff
@@ -34,6 +34,14 @@ export const handle: Handle = async ({ event, resolve }) => {
 			});
 		}
 
+		// Redirect blog to first page
+		if (pathname.endsWith('/blog') || pathname.endsWith('/blog/')) {
+			return new Response(undefined, {
```

Comment:
This is not how you should be doing a redirect here, see: https://kit.svelte.dev/docs/load#redirects

Reply:
Maybe we can also change the one at line 31 https://github.com/wjtje/wjtje.dev/blob/b775b6fa77d226ffd405f17c935b0fdbbb5bd31e/src/hooks.server.ts#L31-L34